src/components/login/loginValidation.js

```diff
@@ -0,0 +1,35 @@
+import Joi from 'joi-browser';
+import { toast } from 'react-toastify';
+
+
+let schema = {
+  email: Joi.string().required().label('Email'),
+  username: Joi.string().required().label('Username'),
+  password: Joi.string().required().label('Password'),
+};
+
+// added this to check
```

Comment:
In my opinion, this would confuse anyone maintaining our codebase in the future.

Reply:
I removed the commented lines.